README.md

```diff
@@ -171,23 +171,22 @@ contain dangerous characters.
 Consider the following ERB template:
 
 ```erb
-<script>
-  var myValue = <%== value.to_json %>;
-</script>
+<textarea>
+  <%== value %>
+</textarea>
 ```
 
-In circumstances where `value` may contain input such as `</script><script>`,
+In circumstances where `value` may contain input such as `</textarea><script>alert(1)</script>`,
 an attacker can easily achieve XSS. We make
 best-effort runtime validations on this value in order to make it safe against
 some obvious attacks.
 
-We wrap the contents of the script tag, including everything between the
-original `<script>` and `</script>`, into a safety check that raises an exception
-if a rogue `</script>` tag is inserted as a result of ruby data being interpolated
-anywhere.
+We check for any interpolation containing `</textarea>` and raise an exception
```

Comment:
the check does not include `>` immediately following the tag since that is not required by browsers
```suggestion
We check for any interpolation containing `</textarea` and raise an exception
```